src/mam4xx/aero_model_emissions.hpp

```diff
@@ -0,0 +1,972 @@
+#ifndef MAM4XX_AERO_MODEL_EMISSIONS_HPP
+#define MAM4XX_AERO_MODEL_EMISSIONS_HPP
+
+#include <haero/atmosphere.hpp>
+
+#include <mam4xx/aero_config.hpp>
+#include <mam4xx/gas_chem_mechanism.hpp>
+#include <mam4xx/mam4_types.hpp>
+#include <mam4xx/utils.hpp>
+
+namespace mam4::aero_model_emissions {
+
+constexpr int pcnst = mam4::pcnst;
+constexpr int n_online_emiss = 9;
```

Comment:
Should we add BAD CONSTANT?

Reply:
I was wondering this, myself, and my thought was that the number and types of tracers in the current implementation of MAM4xx is not configurable, so these hard-coded `int`s are more like fixed model parameters that are only related to online emissions. However, when the day comes that we want to generalize the model, it'd probably be useful to see where numbers are hard-coded.

I'll add in the BAD CONSTANT's , for safety's sake, but am also interested in others' thoughts on the above

Reply:
Should we use another label/comment, for instance, HARD-CODED? @mjs271  @singhbalwinder. I agree that adding this type of comment will help us with moving these parameters to a different place.

Reply:
I do like that idea. For now, I decided to relegate the entire top section to a `BAD_CONSTANTS` namespace since that seemed easier than writing it on every line 😁

Happy to change it to whatever seems best to the group

Reply:
Where is `n_online_emiss` used? That will help us find how it is used.

Reply:
... unused and deleted 🙄

I think it may have been in there from my initial crack at the interface.